CMakeLists.txt

```
@@ -127,6 +127,10 @@ add_cts_option(SYCL_CTS_ENABLE_KHR_QUEUE_EMPTY_QUERY_TESTS
     "Enable extension Khronos queue_empty_query tests" OFF
     FORCE_ON ${SYCL_CTS_ENABLE_KHR_TESTS})
 
+add_cts_option(SYCL_CTS_ENABLE_KHR_MAX_NUM_WORK_GROUPS
```

```suggestion
add_cts_option(SYCL_CTS_ENABLE_KHR_MAX_NUM_WORK_GROUPS_TESTS
```
Actually it looks like `SYCL_CTS_ENABLE_KHR_FREE_FUNCTION_COMMANDS` has also a different naming convention. :thinking: